packages/app/src/cli/services/deploy/upload.ts

```diff
@@ -143,7 +145,9 @@ export async function uploadExtensionsBundle(
   }
 
   const mutation = CreateDeployment
-  const result: CreateDeploymentSchema = await partnersRequest(mutation, options.token, variables)
+  const result: CreateDeploymentSchema = await handlePartnersErrors(() =>
+    partnersRequest(mutation, options.token, variables),
```

Comment:
if you look in the implementation of `partnersRequest`, there is a config: `responseOptions` that accepts an `onError`. 
Can we use that and pass the errorHandler as an optional variable in this function? I think it'd be cleaner and more easy to reuse

Reply:
I only see `onResponse` and `handleErrors`. Am I looking in the wrong place?